tests/src/traffic/steal.rs

```diff
@@ -194,8 +198,10 @@ mod steal {
         let kube_client = kube_client.await;
         let mut flags = vec!["--steal"];
         if let Some(flag) = agent.flag() {
-            flags.extend(flag)
+            flags.extend(flag);
+            Agent::patch_operator(&kube_client).await;
         }
+
```

Comment:
```suggestion
        }
        agent.patch_operator(&kube_client).await;

```
It's a bit ambiguous for the patch to be here I think the better option is to always call the `patch_operator` and let it decide if it needs to be done with `&self` (especially because the patch might flip and we will need to specify when it's not ephemeral)